inline-r/tests/test-qq.hs

```diff
@@ -116,4 +116,13 @@ main = H.withEmbeddedR H.defaultConfig $ H.runRegion $ do
     let utf8string = "abcd çéõßø"
     io . assertEqual "" utf8string =<< fromSEXP <$> R.cast (sing :: R.SSEXPTYPE 'R.String) <$> [r| utf8string_hs |]
 
+
+    -- Disable gctorture, otherwise test takes too long to execute.
```

Comment:
`require(Matrix)` is very heavyweight, it took more than 15 minutes on my system to complete, unfortunately this is the minimal example where I was able to reproduce a problem. If we will have a simpler case it would be OK to remove this.